doc/man7/scrypt.pod

```diff
@@ -86,26 +83,28 @@ This example derives a 64-byte long test vector using scrypt using the password
      assert(!memcmp(out, expected, sizeof(out)));
  }
 
- EVP_PKEY_CTX_free(pctx);
+ EVP_KDF_CTX_free(kctx);
 
 =head1 CONFORMING TO
 
 RFC 7914
 
 =head1 SEE ALSO
 
-L<EVP_PKEY_CTX_set1_scrypt_salt(3)>,
-L<EVP_PKEY_CTX_set_scrypt_N(3)>,
-L<EVP_PKEY_CTX_set_scrypt_r(3)>,
-L<EVP_PKEY_CTX_set_scrypt_p(3)>,
-L<EVP_PKEY_CTX_set_scrypt_maxmem_bytes(3)>,
-L<EVP_PKEY_CTX_new(3)>,
-L<EVP_PKEY_CTX_ctrl_str(3)>,
-L<EVP_PKEY_derive(3)>
+L<EVP_KDF_CTX_set1_pass(3)>,
+L<EVP_KDF_CTX_set1_salt(3)>,
+L<EVP_KDF_CTX_set_scrypt_N(3)>,
+L<EVP_KDF_CTX_set_scrypt_r(3)>,
+L<EVP_KDF_CTX_set_scrypt_p(3)>,
+L<EVP_KDF_CTX_set_scrypt_maxmem_bytes(3)>,
+L<EVP_KDF_CTX_new_id(3)>,
+L<EVP_KDF_CTX_free(3)>,
+L<EVP_KDF_CTX_ctrl_str(3)>,
+L<EVP_KDF_derive(3)>
 
 =head1 COPYRIGHT
 
-Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Uhmmm, that isn't right

Reply:
Oops. Yeah, that started being written as a new file and ended up being almost the same as and replacing the existing file - except now the copyright notice is wrong.
I'll fix that.